### DISCUSSION TITLE
スクリプトについて

### DISCUSSION THREAD

**Discussion by akimotokatsuhiro** [pgroonga/pgroonga#194] [Q&A]
先日、頂いたスクリプトでpgroongaをインストールしたのですが
環境的にインタネットにつながる環境が無く頂いたスクリプトのモジュールをダウンロードして
rpmでインストールしたのですが「依存性の欠如」が出てしまいインストールできませんでした。
このような対処法はありますでしょうか？
以下に出力された一部を記載します。
・arrow-libs-5.0.0.～
　glogはarrow-libs-5.0.0.～に必要とされています
　libre2.so.0()(64bit)はarrow-libs-5.0.0.～に必要とされています
　re2はarrow-libs-5.0.0.～に必要とされています
・groonga-libs-11.0.7～
　libarrow.so.500()(64bit)はgroonga-libs-11.0.7～に必要とされています
   libmsgpack.so.2()(64bit)はgroonga-libs-11.0.7～に必要とされています
   libxxhash.so.0()(64bit)はgroonga-libs-11.0.7～に必要とされています

**Comment by komainu8** [comment]
PGroongaのインストール時に実行したコマンドとエラーメッセージを全てそのまま貼り付けてもらうことはできますか？

少なくともスクリプトでダウンロードしているはずの以下のRPMがインストールされてないように見えます。
* re2-20190801-1.el8.x86_64.rpm
* msgpack-3.1.0-3.el8.x86_64.rpm
* xxhash-libs-0.8.0-3.el8.x86_64.rpm
* glog-0.3.5-3.el8.x86_64.rpm
